pkl-gradle/src/test/kotlin/org/pkl/gradle/EvaluatorsTest.kt

```diff
@@ -501,7 +501,7 @@ class EvaluatorsTest : AbstractTest() {
       }
 
       val printEvalFiles by tasks.registering {
-        inputs.files(doEval)
+        inputs.files(doEval.map { it.effectiveOutputFiles })
```

Comment:
This should not be necessary - it is idiomatic to depend on a task rather than on a specific output property, and also it is not really possible for these properties to be non-empty at the same time, so there is no ambiguity.

Reply:
Fair; will undo this change